qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -0,0 +1,475 @@
+                           docstring='Mode of the output {exp, ramp}.',
+                           get_cmd=partial(self._parent._get_output_function, chan_num),
+                           get_parser=str,
+                           set_cmd=partial(self._parent._set_output_function, chan_num),
+                           set_parser=str
+                           )
+        
+        self.add_parameter('v_range',
+                           label = 'Channel {} voltage range'.format(chan_num),
+                           docstring='Range of the channel in volt.',
+                           set_cmd=partial(self._parent._set_chan_range, chan_num),
+                           set_parser=float,
+                           get_cmd=partial(self._parent._get_chan_range, chan_num),
+                           get_parser=float)
+        
+        self.add_parameter('state',
+                           docstring='State of the channel {on, off}.',
+                           get_cmd=partial(self._parent._get_chan_state, chan_num),
+                           get_parser=bool,
+                           set_cmd=partial(self._parent._set_chan_state, chan_num),
+                           vals=vals.OnOff())
+        
+        self.add_parameter('pos_sat',
+                           get_cmd=partial(self._parent._get_chan_pos_sat, chan_num),
+                           get_parser=str,
+                           set_cmd=partial(self._parent._set_chan_pos_sat, chan_num),
+                           set_parser=str,)
+        
+        self.add_parameter('neg_sat',
+                           get_cmd=partial(self._parent._get_chan_neg_sat, chan_num),
+                           get_parser=str,
+                           set_cmd=partial(self._parent._set_chan_neg_sat, chan_num),
+                           set_parser=str,)
+    
+        self.add_parameter('bilt_name',
+                           set_cmd=partial(self._parent._set_chan_name, chan_num),
+                           set_parser=str)
+
+        self.add_parameter('synchronous',
+                           get_cmd=partial(self._parent._get_chan_synchronous, chan_num),
+                           get_parser=bool,
+                           set_cmd=partial(self._parent._set_chan_synchronous, chan_num),
+                           vals=vals.Bool())
+        
+        
+        self.add_function('start',
+                          call_cmd=lambda: self._parent._set_chan_state(self.chan_num, 1))
+        
+        self.add_function('stop',
+                          call_cmd=lambda: self._parent._set_chan_state(self.chan_num, 0))
+    
+        self.bilt_name(f'Chan{chan_num:02d}')
+
+
+class iTestMultiChannelParameter(MultiChannelInstrumentParameter):
+    """
+    """
+    
+    def __init__(self, channels, param_name, *args, **kwargs):
+        super().__init__(channels, param_name, *args, **kwargs)
+
+    def trigger(self,*args,**kwargs):
+        print(args,kwargs)
```

Comment:
is this really needed? :) having a trigger method that just prints its arguments?)) if it's not needed, it's better to remove in order not to confuse the future readers of this code